scripts/hudson/quickstart.sh

```diff
@@ -156,7 +156,49 @@ function build_apache-karaf {
     exit -1
   fi
 }
+function clone_as {
+  REPO="https://github.com/wildfly/wildfly.git"
+  echo "Cloning AS sources from $REPO"
 
+  cd $WORKSPACE
+
+  if [ -d jboss-as ]; then
+    rm -rf jboss-as # start afresh
+  fi
+
+  git clone $REPO jboss-as || fatal "git clone $REPO failed"
```

Comment:
Maybe check that the clone succeeded here otherwise calling build_as is going to fail.

Reply:
Thanks @mmusgrov for your comment. Doesn't the '||' operator guarantee that either the repo is cloned correctly or the fatal error is thrown?

Reply:
I think as @marcosgopen has it should be OK, we are using `II fatal` elsewhere too in https://github.com/jbosstm/quickstart/blob/master/scripts/hudson/quickstart.sh